packages/tpetra/core/example/Finite-Element-Assembly/fem_assembly_LocalElementLoop_DP.hpp

```diff
@@ -284,8 +295,16 @@ int executeLocalElementLoopDP_(const comm_ptr_t& comm, const struct CmdLineOpts&
       }
 
       // For Type-2 Assembly, we only sumInot the overlapping crs_matrix.
```

Comment:
"sumInto" -- also I don't think the comment applies any more, so you could just delete it.